### PR TITLE
Update API to support NHD

### PR DESCRIPTION
**Pull request by arottersman**
This PR is to continue work on https://github.com/WikiWatershed/rapid-watershed-delineation/pull/36

In addition to @kdeloach's work, it...

- Adds a final re-projection to LatLng to the output shapefiles
- Removes the commit that added a flask debug flag ( [see comment](https://github.com/WikiWatershed/rapid-watershed-delineation/pull/36/files#r85942343) ) 
- Fixes up the initial `rwd-nhd` endpoint commit to specify the NHD data's path into a subdirectory `/nhd`
( [see comment](https://github.com/WikiWatershed/rapid-watershed-delineation/pull/36/files#r85941746) )

### Testing

- Checkout this branch

- Checkout WikiWatershed/docker-rwd#19

If you're on a mac:
- You'll need [Docker for Mac](https://docs.docker.com/engine/installation/mac/), or the Docker toolbox. `docker-machine start` will start a VM called `default`
- I followed the steps in the second answer [here](http://stackoverflow.com/questions/30040708/how-to-mount-local-volumes-in-docker-machine) to give the the `docker-machine` vm access to the RWD data and repo
- Instead of `localhost` you'll use whatever IP `docker-machine ip default` spits out

- From the `docker-rwd` project, run:
```
./scripts/build.sh
RWD_REPO=<where ever you put the repo> RWD_DATA=<where ever you put the data> ./scripts/run.sh
```
- **The RWD NHD data folders need to be in a subdirectory called `nhd`**

Run:

`curl http://localhost:5000/rwd-nhd/39.9526/-75.1652` <= Inside the DRB
`curl http://localhost:5000/rwd/39.9526/-75.1652`  <= Inside the DRB
`curl http://localhost:5000/rwd-nhd/39.0458/-76.6413` <= Outside DRB
`curl http://localhost:5000/rwd/39.0458/-76.6413` <= Outside DRB, should raise appropriate exception

Example output
```
HTTP/1.1 200 OK
Connection: close
Content-Length: 3488
Content-Type: application/json
Date: Tue, 01 Nov 2016 16:04:59 GMT
Server: gunicorn/19.4.5

{
    "input_pt": {
        "features": [
            {
                "geometry": {
                    "coordinates": [
                        -75.16513888892709,
                        39.95337962962963
                    ],
                    "type": "Point"
                },
                "properties": {
                    "Dist_moved": 8,
                    "ID": 1,
                    "Lat": 39.9526,
                    "Lon": -75.1652
                },
                "type": "Feature"
            }
        ],
        "type": "FeatureCollection"
    },
    "watershed": {
        "crs": {
            "properties": {
                "name": "urn:ogc:def:crs:OGC:1.3:CRS84"
            },
            "type": "name"
        },
        "features": [
            {
                "geometry": {
                    "coordinates": [
                        [
                            [
                                -77.52171346605734,
                                42.66530214748923
                            ],
                            ....
                            [
                                -77.52135223128366,
                                42.665250284007094
                            ],
                            
                       ]
                    ],
                    "type": "Polygon"
                },
                "properties": {
                    "Area": 55.1538,
                    "AvgOLF": 0.012716,
                    "BasinLen": 15.537951,
                    "DrnDen": 39.320237,
                    "GRIDCODE": 1,
                    "StrLen": 2168.6605,
                    "Strord": 8.0
                },
                "type": "Feature"
            }
        ],
        "type": "FeatureCollection"
    }
}
```

### CODE REVIEW

File: src/api/main.py [hunk 1]
```diff
@@ -139,8 +139,12 @@ def run_rwd_nhd(lat, lon):
         input_shp_path = os.path.join(output_path, 'New_Outlet.shp')
 
         output = {
-            'watershed': load_json(wshed_shp_path, output_path, simplify),
-            'input_pt': load_json(input_shp_path, output_path)
+            'watershed': load_json(wshed_shp_path, output_path, simplify,
+                                   # From NAD83 / Conus Albers to WGS 84 Latlong
+                                   from_epsg=4269, to_epsg=4326),
```
Comment (mmcfarland) [inline src/api/main.py]: The epsg code which the source data is reprojected from is a different (5070).  Did this new code come from an actual shapeful that was produced from nhd rwd?

https://github.com/WikiWatershed/rapid-watershed-delineation/pull/37/commits/065ee7b309a42161694dd97a33cacac80dcbe666#diff-0179452433a9687f71f6d0aa8e94bcadR116

Comment (arottersman) [inline src/api/main.py]: My bad, I googled NAD83, and got that projection - I'll correct it.

File: src/api/main.py [hunk 2]
```diff
@@ -45,12 +55,11 @@ def run_rwd(lat, lon):
     num_processors = '1'
     data_path = '/opt/rwd-data'
```
Comment (mmcfarland) [inline src/api/main.py]: Let's update this to be `/opt/rwd-data/drb` so that it's a sibling with `/opt/rwd-data/nhd` for the sake of being explicit.

**Comment by arottersman** [comment]
Added a commit that updates the DRB filepath

**Comment by kdeloach** [comment]
Great job!

File: src/api/main.py (hunk 1 above)
Comment (kdeloach) [inline src/api/main.py]: There's a possibility that I used the EPSG code. We should confirm.

**Pull request merged**